quickstart/azure/setup.md

```diff
@@ -92,17 +94,31 @@ For example, a common Service Principal as displayed by the Azure CLI looks some
 }
 ```
 
+However, you also need to obtain subscription id. To get current subscription Id you can use:
+
+```
+az account show --query id -o tsv
+```
+
+To list available subscriptions use:
+
+```
+az account list --query '[].{subscriptionName:name,subscriptionId:id}' -o tsv
+```
+
 The environment variables would then be set as such:
 
 ```bash
-$ export ARM_CLIENT_ID="XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
-$ export ARM_CLIENT_SECRET="YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY"
-$ export ARM_TENANT_ID="ZZZZZZZZ-ZZZZ-ZZZZ-ZZZZ-ZZZZZZZZZZZZ"
+$ export ARM_CLIENT_ID="WWWWWWWW-WWWW-WWWW-WWWW-WWWWWWWWWWWW"
```

Comment:
Mind keeping `ARM_CLIENT_ID` as `XXXX...`, `ARM_CLIENT_SECRET` as `YYYY...`, and `ARM_TENANT_ID` as `ZZZZ` so they match the `pulumi config` commands below?

Reply:
ugh, my bad